### ISSUE TITLE
Navigating back from a root page to a tabbed page causes weird flickering

### ISSUE THREAD

**Issue by jjjhanreyyy** [ionic-team/ionic#13736]
**Ionic version:**  (check one with "x")
(For Ionic 1.x issues, please use https://github.com/ionic-team/ionic-v1)
[ ] **2.x**
[x] **3.x**
[ ] **4.x**

**I'm submitting a ...**  (check one with "x")
[x] bug report
[ ] feature request

Please do not submit support requests or "How to" questions here. Instead, please use one of these channels: https://forum.ionicframework.com/ or http://ionicworldwide.herokuapp.com/

**Current behavior:**

On a Side Menu and Tab app, navigating from a page of tabs to a newly set root page causes a flickering effect when back button is pressed. 
**Expected behavior:**

Hitting back button to return to a tabbed page will flawlessly navigate back to the page.
**Steps to reproduce:**

1. Click Login Button.
2. Open Menu, click Settings page.
3. Hit Browser back button. Do observe URL bar and app header.
**UPDATE**
[Source](https://github.com/jjjhanreyyy/Prosecutor-App)
**Related code:**
**Snippet is inspired by the Ionic Conference Starter App**
```
  openPage(page: PageInterface) {
    let params = {};

    if (page.index) {
      params = { tabIndex: page.index};
    }

    if (this.nav.getActiveChildNavs().length && page.index != undefined) {
        this.nav.getActiveChildNavs()[0].select(page.index);
        console.log('Else executed umdefined');
    } else {
    //I suspect the next line causes the bug. 
    this.app.getRootNav().setRoot(page.pageName, params).catch((err: any) => {
        console.log(`Didn't set nav root: ${err}`);
      });
      console.log('Else executed');
    }
  }
```

**Other information:**

This is how it looks [like](https://www.useloom.com/share/4d3a7bd42b024f16a49716eebcaf7ab0).
[Stackoverflow ](https://stackoverflow.com/questions/47758370/navigating-back-from-a-root-page-to-a-tabbed-page-causes-weird-flickering).
**Ionic info:**

```
    @ionic/cli-utils  : 1.19.0
    ionic (Ionic CLI) : 3.19.0

global packages:

    cordova (Cordova CLI) : 8.0.0

local packages:

    @ionic/app-scripts : 3.1.0
    Cordova Platforms  : none
    Ionic Framework    : ionic-angular 3.9.2

System:

    Node : v8.9.3
    npm  : 5.5.1
    OS   : Windows 10

Environment Variables:

    ANDROID_HOME : not set

Misc:

    backend : pro
```

**Comment by jjjhanreyyy** [comment]
Still no response from the Ionic Team :(

**Comment by kensodemann** [comment]
Hello! Thank you for opening an issue with us!

Given the timing of your opening this issue, there was no one really around to do much responding. Sorry about that, but this is a fairly small team so during the holidays there is going to be slower response on these issues.

I have looked at the video. I don't see this as something that will be fixed in the current version. However, animations have gotten some attention in the next Ionic release. I have marked this as something to look at after that release to see if it is still an issue.

Thank you for using Ionic
